environment.yml

```diff
@@ -25,8 +25,10 @@ dependencies:
   - multidict
   - frozendict
   - nb_conda_kernels
+  - pip
```

Comment:
why is this necessary?

Reply:
Newer versions of Conda raise a warning without this. Unrelated to the rest of the PR.

Reply:
oh, OK, thanks